src/js/utils/ActionDots.jsx

```diff
@@ -38,6 +39,21 @@ const ActionDots = ({
     return actionItemType.BUTTON;
   };
 
+  /** Returns a URL with appended id to provided string
+   * or resolves a function with argument of an id
+   * @param action: { href: {string} | {func}, appendId: boolean | undefined }
+   * @returns {string|null}
+   */
+  const buildLink = (action) => {
+    if (typeof action.href === 'string') {
+      return action.href + (action.appendId === false ? '' : `/${id}`);
+    }
+    if (typeof action.href === 'function') {
+      return action.href(id);
+    }
+    return null;
+  };
+
```

Comment:
while giving the review from the top, I have not seen there.
I won't delete my comments, but they don't make sense anymore seeing this.
Anyway I have mixed feelings if we should some kind of parsing here, or make the links clear for everyone that we don't have to remember: "oh, here I have to use interceptor, and here I don't have to".
If it were up to me, I'd add the context path to the links and forget about `stringUrlInteceptor` and then eventually get rid of the context path if a page becomes a React one.

Reply:
the function itself also is difficult to read for me honestly. By doing it like `action.appendId === false`, wouldn't we have to make sure that in every action defined, we'd have to add `appendId: false`? 
I assume in most places we might not define it at all, so it would be `undefined`, and `undefined !== false` so we would append the id.

Reply:
I think I might have been the one to steer Artur in a direction toward the stringUrlInterceptor back in 2018. If that is so, I'll take the blame. 

I think we can all probably agree on which option below is more readable. Option 1 is more readable, but all probably not the right approach. 
```
href={window.CONTEXT_PATH + '/location/list')}
```
OR
```
href={stringUrlInterceptor(LOCATION_URL.list())}
```
We need to find a way to create URLs that are easy to read without obfuscating the code into oblivion.

What are other projects doing? Has there never been a project that dealt with this issue before? And if the answer is no, then why? If the answer to that is because we are colocating React code with server code in the same repository, then I'm going to vote for moving the React application into its own repository so we don't have to deal with these kinds of issues.

Reply:
@jmiranda I still like having this centralized url factory, and using using "defined" URLs in href. Imho this is a fairly clean and way better approach than the `stringUrlInterceptor`.